### DISCUSSION TITLE
Any way to deploy DART language based server functions with Supabase?

### DISCUSSION THREAD

**Discussion by davoutuk** [supabase/supabase#13372] [General]
I'm building a Flutter based app that will at some stage need some server side functionality.

Is there any way to deploy Dart language based server functions ?    I'm trying to avoid rewriting data models and core functions in another language like TypeScript.

**Comment by dshukertjr** [comment]
Dart Edge might be what you are looking for!
https://supabase.com/docs/guides/functions/dart-edge
